### DISCUSSION TITLE
Getting error "Invalid Version: latest.0.0" in semver

### DISCUSSION THREAD

**Discussion by NitinRamnani** [tauri-apps/tauri#3036] [Q&A] [answered]
Hi,

I'm trying to use tauri with my VueJS app but when I run "npm run tauri info" I get below error. Can some please help me resolving the issue.

TypeError: Invalid Version: latest.0.0

```
[tauri]: running info

Operating System - Windows_NT(10.0.18363) - win32/x64
Microsoft Edge - 44.18362.449.0_neutral__8wekyb3d8bbwe

Node.js environment
  Node.js - 16.13.0
  tauri.js - 0.14.1

Rust environment
  rustc - 1.57.0
  cargo - 1.57.0
```

**Comment by amrbashir** [comment, marked as answer]
you are using a really outdated version of tauri. please follow the instructions on our site to use the latest tauri.
